Samples/Samples.fsproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <GenerateProgramFile>false</GenerateProgramFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NUnit" Version="3.13.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="Samples.fs" />
+    <Compile Include="Program.fs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Vp.FSharp.Sql.PostgreSql\Vp.FSharp.Sql.PostgreSql.fsproj" />
+  </ItemGroup>
```

Comment:
I think we should reference the latest package... hm not sure... it's a tough decision here.

Reply:
Additionally might need a CI/CD + rel. GitHub workflow(also required for unit tests).